src/StackExchange.Redis/PublicAPI/PublicAPI.Unshipped.txt

```diff
@@ -1 +1 @@
-﻿
\ No newline at end of file
+﻿StackExchange.Redis.ChannelMessageQueue.GetAsyncEnumerator(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Collections.Generic.IAsyncEnumerator<StackExchange.Redis.ChannelMessage>!
```

Comment:
Can you please move to the shipped API doc?

Reply:
> Can you please move to the shipped API doc?

done